recipes/zfp/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "zfp" %}
+{% set version = "0.5.5" %}
+{% set build = 0 %}
+{% set sha256 = "fdf7b948bab1f4e5dccfe2c2048fd98c24e417ad8fb8a51ed3463d04147393c5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://computing.llnl.gov/projects/floating-point-compression/download/zfp-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  run_exports:
+    # patch-releases are not ABI compatible
+    - {{ pin_subpackage('zfp', min_pin='x.x.x', max_pin='x.x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make          # [unix]
+    - cmake >=3.4.0
+  # a variant could provide CUDA support
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/cmake/zfp/zfp-config.cmake                      # [unix]
+    - test -f ${PREFIX}/lib/libzfp.so                                       # [linux]
+    - otool -L ${PREFIX}/lib/libzfp.dylib                                   # [osx]
+    - if exist %LIBRARY_PREFIX%\bin\libzfp.dll (exit 0) else (exit 1)       # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\zfp\zfp-config.cmake (exit 0) else (exit 1)  # [win]
+
+about:
+  home: http://computing.llnl.gov/projects/floating-point-compression
+  license: BSD-3-Clause-LLNL
```

Comment:
Dunno what to put here, it's not a plain `BSD-3-Clause` but a variant such as `BSD-3-Clause-LBNL` and `BSD-3-Clause-UC`: https://spdx.org/licenses/

Reply:
Yeah, there is some way to do custom licenses

Reply:
See https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions
`LicenseRef-*` is valid

Reply:
According to some LLNL folks, this can be treated as BSD-3-Clause.
I'll wait for @lindstro to confirm this, otherwise will use `LicenseRef-LLNL` or `LicenseRef-ZFP` or so...